docs/getting_started.rst

```diff
@@ -66,6 +66,11 @@ Installation
 .. note::
    The server and client versions of nvflare must match, we do not support cross-version compatibility.
 
+Supported Operating Systems
+---------------------------
+- Linux
+- OSX (Note: some dependencies are not compatible)
```

Comment:
I think it might be better to specifically points out what dependencies (something in app-opt?) are not compatible